host/onebox.go

```diff
@@ -423,6 +423,8 @@ func (c *temporalImpl) startFrontend(hosts map[string][]string, startWG *sync.Wa
 		fx.Provide(func() authorization.JWTAudienceMapper { return nil }),
 		fx.Provide(func() client.FactoryProvider { return client.NewFactoryProvider() }),
 		fx.Provide(func() searchattribute.Mapper { return nil }),
+		// Comment the line above and uncomment the line bellow to test with search attributes mapper.
+		// fx.Provide(func() searchattribute.Mapper { return NewSearchAttributeTestMapper() }),
```

Comment:
any reason it cannot test the sa mapper by default?

Reply:
This will require rewrite of all tests to use aliases instead of actual search attribute field names. Ideally, I would create another test suite and setup another cluster there but this requires massive integration tests refactoring.